<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="ant_wgt" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="an_lb">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>250</height>
         </size>
        </property>
        <property name="text">
         <string>动画标签</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="left_wgt" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <widget class="QPushButton" name="r_btn">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>200</y>
           <width>80</width>
           <height>23</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	font: 15px &quot;微软雅黑 Light&quot;;
	font-weight:500;
}
</string>
         </property>
         <property name="text">
          <string>立即注册</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="topMargin">
          <number>20</number>
         </property>
         <item row="1" column="0" colspan="2">
          <widget class="QLineEdit" name="pw_le">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    border:none;
	background: transparent;
	border-bottom:2px solid rgb(138, 138, 138);
	
	font: 9pt &quot;微软雅黑&quot;;
    
   
}
QLineEdit:hover{
    border-bottom:2px solid rgb(95, 95, 95);
}
QLineEdit:focus{
    border-bottom:2px solid rgb(82, 209, 255);
}</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="rp_cbx">
           <property name="text">
            <string>记住密码</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QComboBox" name="user_cmbx">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">
QComboBox {
	border:none;
	background: transparent;
	border-bottom:2px solid rgb(138, 138, 138);
    selection-background-color: orange;
	
	font: 9pt &quot;微软雅黑&quot;;
}
QComboBox::drop-down {
	background-color: transparent;
	width: 30px;
    height: 30px;
} 
 
QComboBox:down-arrow {
    image: url(:/l_1/resource/img/bottom_btn.png);
    
} 

QComboBox:hover{
    border-bottom:2px solid rgb(95, 95, 95);
}
QComboBox:focus{
    border-bottom:2px solid rgb(82, 209, 255);
}</string>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>123465</string>
            </property>
            <property name="icon">
             <iconset resource="../../r_l_c.qrc">
              <normaloff>:/l_1/resource/img/safe1.png</normaloff>:/l_1/resource/img/safe1.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>7897987</string>
            </property>
            <property name="icon">
             <iconset resource="../../r_l_c.qrc">
              <normaloff>:/l_1/resource/img/safe2.png</normaloff>:/l_1/resource/img/safe2.png</iconset>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="1" alignment="Qt::AlignRight">
          <widget class="QCheckBox" name="alg_cbx">
           <property name="text">
            <string>自动登录</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QPushButton" name="lg_btn">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
    
    border-radius: 10px;
	
	background-color: rgb(0, 0, 127);
	
	color: rgb(255, 255, 255);
    
    
}
QPushButton:disabled{
    
    color:gray;
	background-color: rgb(98, 92, 127);
    
    
}
QPushButton:hover{
    
    
	background-color: rgb(0, 0, 184);
    
    
}</string>
           </property>
           <property name="text">
            <string>安全登录</string>
           </property>
           <property name="icon">
            <iconset resource="../../r_l_c.qrc">
             <normaloff>:/l_1/resource/img/captain_america.png</normaloff>:/l_1/resource/img/captain_america.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="right_wgt" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <widget class="QPushButton" name="url_btn">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>16</x>
           <y>120</y>
           <width>100</width>
           <height>100</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/l_1/resource/img/bd.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../r_l_c.qrc"/>
 </resources>
 <connections/>
</ui>
